src/components/DropzoneArea.js

```diff
@@ -1,15 +1,13 @@
 import Snackbar from '@material-ui/core/Snackbar';
 import Typography from '@material-ui/core/Typography';
 import {withStyles} from '@material-ui/core/styles';
-import CloudUploadIcon from '@material-ui/icons/CloudUpload';
+import {AttachFile, CloudUpload} from '@material-ui/icons/';
```

Comment:
Avoid spread imports, prefer single ones

```jsx
import AttachFile from '@material-ui/icons/AttachFile';
import CloudUpload from '@material-ui/icons/CloudUpload';
```

Reply:
Yes no problem, I saw a mixture of the 2 in the project and wasn't sure which way was the one we were aiming for.